server/src/main/java/org/candlepin/policy/js/compliance/ComplianceRules.java

```diff
@@ -164,7 +164,7 @@ public ComplianceStatus getStatus(Consumer c, Date date, boolean calculateCompli
 
                 if (updateConsumer && (complianceChanged || entStatusChanged)) {
                     // Merge might work better here, but we use update in other places for this
-                    consumerCurator.update(c);
+                    consumerCurator.updateNoFlush(c);
```

Comment:
Can we change this to instead overload the update method to take a boolean value specifying whether or not we want to flush, then default the one-param version to "true" ?